app/Http/Controllers/Settings/ApiTokenController.php

```diff
@@ -19,13 +19,11 @@ public function __construct()
 
     public function store(CreateApiTokenRequest $request): RedirectResponse
     {
-        $this->dispatchSync(new CreateApiToken($user = $request->user(), $request->name()));
-
-        $token = $user->tokens()->where('name', $request->name())->first();
+        $plainTextToken = $this->dispatchSync(new CreateApiToken($request->user(), $request->name()));
 
         $this->success('settings.api_token.created');
 
-        return redirect()->route('settings.profile')->with('api_token', $token->plainTextToken);
+        return redirect()->route('settings.profile')->with('api_token', $plainTextToken);
```

Comment:
Can you revert all these changes about the token? They should already be on master.